SK-2009.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "@SK-2009"
+project: "Spy Camera"
+---
+
+# Project Name
+Mini Security Camera
+## Summary
+
+What I plan on building is a small camera in a 3D printed enclosure that I can place anywhere with a adhesive bottom and go unnoticed for a long time
+
+## Plan
+
+1. Design and print enclosure, Solder boards together, Connect anntennas to the boards, Mount camera and other components, Plug in battery, Watch on my analog screen
```

Comment:
when you say solder boards together, which boards are you referring to?

Reply:
> when you say solder boards together, which boards are you referring to?

The VTX and the camera, It might be able to plug in but I also might just need to get a small cable between the two

Reply:
ahh cool! could you please edit your file and swap out "solder boards together" for "connect the VTX and the camera" or some variation of that? also, could you please look into whether you'll need a cable so that you can add it in your budget and purchase it hehe :)

thank you sm!